src/auth/redis/redis.module.ts

```diff
@@ -0,0 +1,23 @@
+import { Module } from '@nestjs/common';
+import { RedisService } from './redis.service';
+import { CacheModule } from '@nestjs/cache-manager';
+import { ConfigService } from '@nestjs/config';
+import * as redisStore from 'cache-manager-redis-store';
+
+@Module({
+  imports: [
+    CacheModule.registerAsync({
+      useFactory: async (configService: ConfigService) => ({
+        store: redisStore,
+        host: configService.get<string>('REDIS_HOST'),
+        port: configService.get<number>('REDIS_PORT'),
+        password: configService.get<string>('REDIS_PASSWORD'),
+        ttl: Number(configService.get<number>('REDIS_EXPIRATION_TIME')),
```

Comment:
```suggestion
        ttl: Number(configService.get<string>('REDIS_EXPIRATION_TIME')),
```
env에서 REDIS_EXPIRATION_TIME을 string타입으로 가져오기 때문에 string으로 명시해줘야 오해가 없을거 같습니다.

Reply:
맞네요 어차피 스트링타입으로 가져오니까 그렇게 변경해야겠습니다~